crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -2,6 +2,11 @@
 //!
 //! An **entity** exclusively owns zero or more [component] instances, all of different types, and can dynamically acquire or lose them over its lifetime.
 //!
+//! **empty entity**: Entity ownes zero [component] is called empty entity, it has a valid [`EntityLocation`] which points to [`ArchetypeId::EMPTY`]
+//! **pending entity**: Entity reserved, but not flushed yet([`Entities::flush`] docs for reference).
```

Comment:
```suggestion
//! **pending entity**: Entity reserved, but not flushed yet(see [`Entities::flush`] docs for reference).
```

Reply:
Also please add a space before the parentheses.